src/mudlet-lua/tests/DB_spec.lua

```diff
@@ -1,28 +1,8 @@
 describe("Tests DB.lua functions", function()
-  setup(function()
-    -- add in the location of our files
-    package.path = "../lua/?.lua;"
-
-    -- add in the location of Lua libraries on Ubuntu 12.04
-    package.path = package.path .. "/usr/local/share/lua/5.1/?.lua;/usr/local/share/lua/5.1/?/init.lua;/usr/local/lib/lua/5.1/?.lua;/usr/local/lib/lua/5.1/?/init.lua;/usr/share/lua/5.1/?.lua;/usr/share/lua/5.1/?/init.lua"
-
-    luasql = require"luasql.sqlite3"
-    rex    = require"rex_pcre"
-
-    -- define some common Mudlet functions essential to operation
-    function getMudletHomeDir() return "." end
-
-    require"TableUtils"
-  end)
-
-  it("Should load DB.lua", function()
-    require"DB"
-    assert.truthy(db)
-  end)
 
   describe("Tests that DB creation and deletion works", function()
     it("Should create a db", function()
-      mydb = db:create("people", {
+      mydb = db:create("peopletguw", {
```

Comment:
What does the `guw` suffix mean? It's not intuitive

Reply:
Yeah, that is puzzling me as well... 💭

Reply:
It's random letters, to avoid name collisions with existing DBs. At first I had 'mtf' for mudlet test framework, but then I thought, "what if someone has a DB named people_mtf that stands for 'people my top friends'..." So I used random letters that didn't look like they could mean anything.

Reply:
What do you think of this one?

Reply:
That'll work. I've updated the other stuff required for the DB name changes and it should be good to go.